### ISSUE TITLE
Cupertino Icons draws the wrong icon that you selected in the code

### ISSUE THREAD

**Issue by gorkovv** [flutter/flutter#72237]
## Steps to Reproduce

Selecting an icon from the library CupertinoIcons
`Icon(CupertinoIcons.text_cursor, size: 16)`
The icon is displayed correctly in the code editor:

![Снимок экрана от 2020-12-13 13-11-42](https://user-images.githubusercontent.com/7299332/102012880-4c966e00-3d7f-11eb-9ee1-01cc41fd0393.png)

In the app, the icon is not the same, but the one next to it.

![Снимок экрана от 2020-12-13 20-01-09](https://user-images.githubusercontent.com/7299332/102013054-405ee080-3d80-11eb-895b-bf4def4257d7.png)

![Снимок экрана от 2020-12-13 19-14-19](https://user-images.githubusercontent.com/7299332/102013070-54a2dd80-3d80-11eb-9982-836aaaa29bb8.png)

If I choose in the code  `CupertinoIcons.text_bubble_fill` then I'll get it in the app  `CupertinoIcons.text_cursor`

But not all icons behave this way. Some are displayed correctly.

Library version:
`cupertino_icons: ^1.0.1+1`

<details>
  <summary>Logs</summary>

##flutter doctor -v

[✓] Flutter (Channel dev, 1.25.0-8.0.pre, on Linux, locale ru_RU.UTF-8)
    • Flutter version 1.25.0-8.0.pre at /home/gorkov/flutter
    • Framework revision a12e2a473a (4 дня назад), 2020-12-08 23:13:04 -0500
    • Engine revision df39e5c515
    • Dart version 2.12.0 (build 2.12.0-133.0.dev)

[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.2)
    • Android SDK at /home/gorkov/Android/Sdk
    • Platform android-29, build-tools 29.0.2
    • Java binary at: /home/gorkov/android-studio/jre/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6222593)
    • All Android licenses accepted.

[✓] Android Studio
    • Android Studio at /home/gorkov/android-studio
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6222593)

[✓] VS Code (version 1.52.0)
    • VS Code at /usr/share/code
    • Flutter extension version 3.17.0

[✓] Connected device (1 available)
    • Android SDK built for x86 (mobile) • emulator-5554 • android-x86 • Android 7.0 (API 24)
      (emulator)

• No issues found!

</details>

**Comment by ZainUrRehmanKhan** [comment]
No problem on my side, working totally fine.

**Comment by yash1200** [comment]
Reproducible on latest master.
My flutter doctor
```
[✓] Flutter (Channel master, 1.25.0-9.0.pre.42, on Linux, locale en_IN.utf8)
    • Flutter version 1.25.0-9.0.pre.42 at /home/yash/flutter
    • Framework revision a76bb1a08e (32 hours ago), 2020-12-12 00:17:51 -0700
    • Engine revision 4797b06652
    • Dart version 2.12.0 (build 2.12.0-141.0.dev)

[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.2)
    • Android SDK at /home/yash/Android/Sdk
    • Platform android-30, build-tools 30.0.2
    • Java binary at: /home/yash/android-studio/jre/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6222593)
    • All Android licenses accepted.

[✓] Chrome - develop for the web
    • CHROME_EXECUTABLE = /usr/bin/google-chrome-beta

[✓] Linux toolchain - develop for Linux desktop
    • clang version 11.0.0
    • cmake version 3.19.1
    • ninja version 1.10.1
    • pkg-config version 1.7.3

[✓] Android Studio
    • Android Studio at /home/yash/android-studio
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6222593)

[✓] Connected device (2 available)
    • Linux (desktop) • linux  • linux-x64      • Linux
    • Chrome (web)    • chrome • web-javascript • Google Chrome 87.0.4280.66 beta
```

**Comment by Pr47h4m** [comment]
I'm also facing same issue 
i have upgraded my flutter today

**Comment by TahaTesser** [comment]
Hey everyone
Even tho it seems to be tied to `cupertino_icons`, the issue seems to be reproduce using `cupertino_icons: ^1.0.0` as well and seems related to Flutter itself 
It can correctly shows text_cursor on `stable` & `beta and regression on `dev` and `master`

### Code
```dart
      body: Center(
        child: Icon(CupertinoIcons.text_cursor),
      ),
```

| stable & beta | dev & master |
| --------------- | --------------- |
![Screenshot_1607952524](https://user-images.githubusercontent.com/48603081/102087173-2a642500-3e3f-11eb-9b80-8f0d0958e35a.png) | ![Screenshot_1607952585](https://user-images.githubusercontent.com/48603081/102087181-2d5f1580-3e3f-11eb-96ab-b7ffd5a57ab7.png)


<details> 
<summary>flutter doctor -v</summary> 

```bash
[✓] Flutter (Channel dev, 1.25.0-8.0.pre, on Microsoft Windows [Version 10.0.19042.685], locale en-US)
    • Flutter version 1.25.0-8.0.pre at C:\Code\flutter_dev
    • Framework revision a12e2a473a (5 days ago), 2020-12-08 23:13:04 -0500
    • Engine revision df39e5c515
    • Dart version 2.12.0 (build 2.12.0-133.0.dev)

[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.3)
    • Android SDK at C:\Code\sdk
    • Platform android-30, build-tools 30.0.3
    • ANDROID_HOME = C:\Code\sdk
    • Java binary at: C:\Code\android-studio\jre\bin\java
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b01)
    • All Android licenses accepted.

[✓] Chrome - develop for the web
    • Chrome at C:\Program Files\Google\Chrome\Application\chrome.exe

[✓] Visual Studio - develop for Windows (Visual Studio Community 2019 16.8.3)
    • Visual Studio at C:\Program Files (x86)\Microsoft Visual Studio\2019\Community
    • Visual Studio Community 2019 version 16.8.30804.86
    • Windows 10 SDK version 10.0.18362.0

[✓] Android Studio (version 4.1.0)
    • Android Studio at C:\Code\android-studio
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b01)

[✓] VS Code (version 1.52.0)
    • VS Code at C:\Users\Taha\AppData\Local\Programs\Microsoft VS Code
    • Flutter extension version 3.17.0

[✓] Connected device (4 available)
    • sdk gphone x86 (mobile) • emulator-5554 • android-x86    • Android 11 (API 30) (emulator)
    • Windows (desktop)       • windows       • windows-x64    • Microsoft Windows [Version 10.0.19042.685]
    • Chrome (web)            • chrome        • web-javascript • Google Chrome 87.0.4280.88
    • Edge (web)              • edge          • web-javascript • Microsoft Edge 85.0.564.67

• No issues found!
```

</details>

**Comment by HansMuller** [comment]
CC @xster @willlarche

**Comment by huage2580** [comment]
the same. in master channel

**Comment by willlarche** [comment]
Has anyone made changes to these lately?

**Comment by xster** [comment]
@guidezpl I'm going to [revert](https://github.com/flutter/flutter/pull/72358) https://github.com/flutter/flutter/pull/71853 unfortunately. It seems like it shifted the glyph codepoints forward by one in addition to the doc changes.

**Comment by xster** [comment]
Strange however that the generate script wasn't materially changed in the PR that generated the new dart file. Taking a look.

**Comment by xster** [comment]
Ok, fixing forward rather. There was some tricky nuance I just figured out. Between the triple dance of flutter/flutter, flutter/cupertino_icons and xster/framework7-icons where everything's generated, flutter/flutter and flutter/cupertino_icons are up to date. Except I forgot to push the last commit from xster/framework7-icons that I used to generate flutter/flutter and flutter/cupertino_icons with onto GitHube. When @guidezpl updated xster/framework7-icons, it was off of HEAD~1 and all the codepoints were misaligned by 1. 

Fixing in https://github.com/flutter/flutter/pull/72384 (and adding some more tests). The missing bit was https://github.com/xster/framework7-icons/commit/2e486f3dfe8aafcca677410b6b1e504794b0ea14.
